src/ontology/mondo-edit.obo

```diff
@@ -165479,8 +165479,8 @@ intersection_of: MONDO:0001347 ! facioscapulohumeral muscular dystrophy
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/29090 ! SMCHD1
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 ! DUX4
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/29090 {source="MONDO:mim2gene_medgen", source="OMIM:158901"} ! SMCHD1
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 {source="OMIM:158901"} ! DUX4
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/29090 {source="MONDO:mim2gene_medgen"} ! SMCHD1
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 ! DUX4
```

Comment:
These gene associations are for MONDO:0008031 'facioscapulohumeral muscular dystrophy 2'.

[OMIM:158901](https://omim.org/entry/158901) also has "digenic" in the disease description.

Reply:
See my tentative explanation above https://github.com/monarch-initiative/mondo/pull/8624#discussion_r1931113562

If it is the case that these are, perhaps erroneously labeled as digenic, but otherwise meet all of the logical conditions for which we would normally add a disease defining association, do you want me to add an explicit filter to remove anything that happens to have digenic in the label?

Reply:
Since MONDO:0008031 is in the [exclusion file](https://github.com/monarch-initiative/omim/blob/main/data/exclusions-disease-gene.tsv) a gene association was not intended to be added. It has now been manually removed.